projects/detr3d/configs/detr3d_vovnet_gridmask_det_final_trainval_cbgs_dev-1.x_test_old.py

```diff
@@ -0,0 +1,243 @@
+_base_ = [
+    # '.../mmdetection3d/configs/_base_/datasets/nus-3d.py',
+    '../../../configs/_base_/default_runtime.py'
+]
+
+custom_imports = dict(imports=['projects.detr3d.mmdet3d_plugin'])
+# If point cloud range is changed, the models should also change their point
+# cloud range accordingly
+point_cloud_range = [-51.2, -51.2, -5.0, 51.2, 51.2, 3.0]
+voxel_size = [0.2, 0.2, 8]
+
+img_norm_cfg = dict(mean=[103.530, 116.280, 123.675],
+                    std=[57.375, 57.120, 58.395],
+                    bgr_to_rgb=False)
+# For nuScenes we usually do 10-class detection
+class_names = [
+    'car', 'truck', 'construction_vehicle', 'bus', 'trailer', 'barrier',
+    'motorcycle', 'bicycle', 'pedestrian', 'traffic_cone'
+]
+
+input_modality = dict(use_lidar=False,
+                      use_camera=True,
+                      use_radar=False,
+                      use_map=False,
+                      use_external=False)
+# this means type='Detr3D' will be processed as 'mmdet3d.Detr3D'
+default_scope = 'mmdet3d'
```

Comment:
Please remove the `final` word  in this file name. Also, this config can inherit from the `projects/detr3d/configs/detr3d_res101_gridmask_cbgs_dev-1.x_test_old.py` since there are only few differences.